### ISSUE TITLE
Offline XHR fails if SW is newly activated.

### ISSUE THREAD

**Issue by paullewis** [GoogleChrome/lighthouse#179]
Not at all sure why.

Steps for repro:

1. Unregister SW in chrome://serviceworker-internals for https://voice-memos.appspot.com/
2. Hit https://voice-memos.appspot.com/ (note prompt that app is now ready for offline)
3. Run extension, see that XHR request fails
4. Refresh
5. Run extension again, see that XHR request succeeds

**Comment by brendankenny** [comment]
cc @ebidel 
Just ran into this in the CLI for https://events.google.com/io2016/ but it appears to behave the same with voice-memos under the CLI.

Once you unregister the SW and run with the CLI, you get
```
Works Offline: 0%
 -- Has a registered Service Worker: false
 -- URL responds with a 200 when offline: false
```
if you then rerun the CLI (with or without restarting Chrome), you'll continue to get false on both of these. Only manually refreshing or navigating to the page seems to correct the issue, after which both sites will get 
```
Works Offline: 100%
 -- Has a registered Service Worker: true
 -- URL responds with a 200 when offline: true
```

**Comment by paullewis** [comment]
@samccone this should be fixed by #187, right?

**Comment by samccone** [comment]
indeed
